tools/argdist.py

```diff
@@ -611,6 +611,8 @@ def __init__(self):
                   help="maximum string size to read from char* arguments")
                 parser.add_argument("-i", "--interval", default=1, type=int,
                   help="output interval, in seconds")
+                parser.add_argument("-d", "--duration",
+                  help="total duration of trace, seconds")
```

Comment:
What do you think on adding here the type, so we don't have to cast it below? (also applies to the other files :) )

Reply:
Thanks. I have addressed it in new commit.